terraform/layer2-k8s/templates/prometheus-values.tmpl

```diff
@@ -125,13 +125,25 @@ grafana:
   dashboards:
     logs:
       logs:
-        url: https://gist.githubusercontent.com/khalezin/49766ed2976d49b0dc84a9c7809392e5/raw/53c0d067713e4da0e6cf187c6515a19e5dd996f5/loki-dashboard-quick-search.json
+        url: https://s3.amazonaws.com/grafana-dashboards.maddevs.org/common/aws-eks-base/loki-dashboard-quick-search.json
+
     k8s:
       nginx-ingress:
```

Comment:
can you please, add the link to dashboard homepage as a comment

Reply:
I added short description with link to dashboard homepage.